### PR TITLE
1. Single Page Applicatione.

### PR DESCRIPTION
**Pull request by olga-kravchenko**

### CODE REVIEW

File: src/view/trip-events-point.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+export const createTripPointTemplate = () => {
+  return ` <li class="trip-events__item">
```
Comment (rmk1409) [inline src/view/trip-events-point.js:2]: Remove extra space.

File: src/main.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import {createCostOfTripTemplate} from "./view/cost-of-trip.js";
+import {createTripTabsTemplate} from "./view/trip-tabs.js";
+import {createTripFiltersTemplate} from "./view/trip-filters.js";
+
+const render = (container, template, place) => {
+  container.insertAdjacentHTML(place, template);
+};
+
+const headerSection = document.querySelector(`.trip-main`);
+render(headerSection, createTripInfoTemplate(), `afterbegin`);
```
Comment (rmk1409) [inline src/main.js:16]: Save order for comments.

File: src/main.js [hunk 3]
```diff
@@ -18,3 +23,16 @@ render(tripControlsFirstTittle, createTripTabsTemplate(), `afterend`);
...
+const tripEvents = document.querySelector(`.trip-events`);
+const tripEventsTittle = tripEvents.querySelector(`.trip-events h2`);
+render(tripEventsTittle, createTripSortTemplate(), `afterend`);
+render(tripEvents, createTripEventsListTemplate(), `beforeend`);
+
+const tripEventsList = tripEvents.querySelector(`.trip-events__list`);
+render(tripEventsList, createFormOfCreatingPointTemplate(), `afterbegin`);
+render(tripEventsList, createEditPointTemplate(), `afterbegin`);
+
+for (let i = 0; i < 3; i++) {
```
Comment (rmk1409) [inline src/main.js:36]: Get rid of MAGIC NUMBER.

File: src/main.js [hunk 4]
```diff
@@ -2,6 +2,11 @@ import {createTripInfoTemplate} from "./view/trip-info.js";
 import {createCostOfTripTemplate} from "./view/cost-of-trip.js";
 import {createTripTabsTemplate} from "./view/trip-tabs.js";
 import {createTripFiltersTemplate} from "./view/trip-filters.js";
+import {createTripSortTemplate} from "./view/trip-sort.js";
```
Comment (rmk1409) [inline src/main.js:5]: Make an order in file/function namings.

File: package.json [hunk 5]
```diff
@@ -5,11 +5,14 @@
   "description": "Проект «Большое путешествие» от HTML Academy",
   "scripts": {
     "eslint": "eslint src/**/*.js",
-    "test": "npm run eslint"
+    "test": "npm run eslint",
+    "build": "webpack --mode development"
```
[the remark this answers is not in the document]
Comment (olga-kravchenko) [reply package.json]: fixed

File: .gitignore [hunk 6]
```diff
@@ -11,3 +11,4 @@ node_modules/
 
 # Собранные файлы
```
[the remark this answers is not in the document]
Comment (olga-kravchenko) [reply .gitignore:12]: Fixed

File: package.json [hunk 7]
```diff
@@ -5,11 +5,14 @@
...
   },
   "devDependencies": {
     "eslint": "7.13.0",
-    "eslint-config-htmlacademy": "0.6.0"
+    "eslint-config-htmlacademy": "0.6.0",
+    "webpack": "^4.44.2",
```
[the remark this answers is not in the document]
Comment (olga-kravchenko) [reply package.json:15]: Fixed

File: markup/add-new-point.html [hunk 8]
```diff
@@ -1,4 +1,3 @@
-<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (olga-kravchenko) [reply markup/add-new-point.html:1]: Fixed

File: src/view/cost-of-trip.js [hunk 9]
```diff
@@ -0,0 +1,6 @@
+export const createCostOfTripTemplate = () => {
```
[the remark this answers is not in the document]
Comment (olga-kravchenko) [reply src/view/cost-of-trip.js:1]: Fixed

File: src/view/trip-events-point.js (hunk 1 above)
Comment (olga-kravchenko) [reply src/view/trip-events-point.js:2]: Fixed

File: src/main.js (hunk 2 above)
Comment (olga-kravchenko) [reply src/main.js:16]: Fixed

File: src/main.js (hunk 3 above)
Comment (olga-kravchenko) [reply src/main.js:36]: Fixed

File: src/main.js (hunk 4 above)
Comment (olga-kravchenko) [reply src/main.js:5]: Fixed

File: package.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (rmk1409) [reply package.json:15]: Not fixed.

File: .gitignore (hunk 6 above)
[the remark this answers is not in the document]
Comment (rmk1409) [reply .gitignore]: Refactor.

**Comment by rmk1409** [comment]
Rename **trip-list**.

**Comment by rmk1409** [comment]
Simplify names.

File: .gitignore (hunk 6 above)
[the remark this answers is not in the document]
Comment (olga-kravchenko) [reply .gitignore]: Fixed

File: package.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (olga-kravchenko) [reply package.json:15]: fixed

**Pull request merged**